src/dataflow/load.js

```diff
@@ -43,7 +43,7 @@ export function request(target, url, format) {
         df.ingest(target, data, format);
       },
       function(error) {
-        df.error('Loading failed: ' + url, error);
+        df.error('Loading failed: ' + (typeof url === 'string' ? url : JSON.stringify(url)), error);
```

Comment:
In that case, I would prefer that we simply do this, which works well with the default `console.error` output and is more consistent with error handling elsewhere.

```js
df.error('Loading failed', url, error);
```